### PR TITLE
Added possibility to define multiple choice field in custom fields

### PR DESCRIPTION
**Pull request by kkrampa**
@esoergel @czue 
This PR that contains only changes related with core code.

@amsagoff

### CODE REVIEW

**Comment by kkrampa** [comment]
@esoergel I made changes according to your comments from #7279

File: corehq/apps/custom_data_fields/templates/custom_data_fields/custom_data_fields.html [hunk 1]
```diff
@@ -159,6 +159,12 @@
                                             </a>
                                         </li>
                                     </ul>
+                                    {% if multiple_choice_field_enabled %}
```
Comment (esoergel) [inline corehq/apps/custom_data_fields/templates/custom_data_fields/custom_data_fields.html]: you can instead do
```
{% if request|toggle_enabled:"MULTIPLE_CHOICE_CUSTOM_FIELD" %}
```
and avoid passing parameters to the view

**Comment by esoergel** [comment]
One small comment, but it looks good overall!

**Comment by esoergel** [comment]
@kkrampa looks like there's a merge conflict - can you merge in master and address my last comment please?  I'll be happy to merge this PR after that.

**Comment by kkrampa** [comment]
@esoergel Done

**Pull request merged**